extracters/aata_data.py

```diff
@@ -48,18 +48,35 @@ def language_object_from_code(code):
 	'''
 	languages = {
 		# TODO: there must be a better way to do this than keep a static mapping of languages
+		'cze': {'ident': 'http://vocab.getty.edu/aat/300388191', 'label': 'Czech'},
+		'dan': {'ident': 'http://vocab.getty.edu/aat/300388204', 'label': 'Danish'},
+		'nld': {'ident': 'http://vocab.getty.edu/aat/300388256', 'label': 'Dutch'},
+		'dut': {'ident': 'http://vocab.getty.edu/aat/300388256', 'label': 'Dutch'},
 		'eng': {'ident': 'http://vocab.getty.edu/aat/300388277', 'label': 'English'},
+		'gre': {'ident': 'http://vocab.getty.edu/aat/300388361', 'label': 'Greek'}, # TODO: the 'gre' ISO code is for modern Greek; confirm that that's what is being represented here
 		'fra': {'ident': 'http://vocab.getty.edu/aat/300388306', 'label': 'French'},
 		'fre': {'ident': 'http://vocab.getty.edu/aat/300388306', 'label': 'French'},
+		'geo': {'ident': 'http://vocab.getty.edu/aat/300388343', 'label': 'Georgian'},
 		'ger': {'ident': 'http://vocab.getty.edu/aat/300388344', 'label': 'German'},
 		'deu': {'ident': 'http://vocab.getty.edu/aat/300388344', 'label': 'German'},
-		'pol': {'ident': 'http://vocab.getty.edu/aat/300389109', 'label': 'Polish'},
+		'heb': {'ident': 'http://vocab.getty.edu/aat/300388401', 'label': 'Hebrew'},
+		'hun': {'ident': 'http://vocab.getty.edu/aat/300388770', 'label': 'Magyar (Hungarian) '},
 		'ita': {'ident': 'http://vocab.getty.edu/aat/300388474', 'label': 'Italian'},
-		'nld': {'ident': 'http://vocab.getty.edu/aat/300388256', 'label': 'Dutch'},
+		'jpn': {'ident': 'http://vocab.getty.edu/aat/300388486', 'label': 'Japanese'},
+		'lat': {'ident': 'http://vocab.getty.edu/aat/300388693', 'label': 'Latin'},
+		'nor': {'ident': 'http://vocab.getty.edu/aat/300388992', 'label': 'Norwegian'},
+		'pol': {'ident': 'http://vocab.getty.edu/aat/300389109', 'label': 'Polish'},
+		'por': {'ident': 'http://vocab.getty.edu/aat/300389115', 'label': 'Portuguese'},
+		'rus': {'ident': 'http://vocab.getty.edu/aat/300389168', 'label': 'Russian'},
+		'slo': {'ident': 'http://vocab.getty.edu/aat/300389290', 'label': 'Slovak'},
 		'spa': {'ident': 'http://vocab.getty.edu/aat/300389311', 'label': 'Spanish'},
-		'swe': {'ident': 'http://vocab.getty.edu/aat/300389336', 'label': 'Swedish'}
+		'swe': {'ident': 'http://vocab.getty.edu/aat/300389336', 'label': 'Swedish'},
+		'tur': {'ident': 'http://vocab.getty.edu/aat/300389470', 'label': 'Turkish'},
+		# TODO: rom; is this Romany or maybe (likely) Romanian?
 	}
```

Comment:
Future to do is still to figure out how to manage this sort of thing across projects :) e.g. this /could/ be `'ita': vocab.instances['italian language']` but is there an advantage to doing that .... probably not today

Reply:
Agreed.